README.md

```diff
@@ -22,4 +22,32 @@ A Snap plugin is a program that responds to a set of well defined [gRPC](http://
 You will find [example plugins](examples) that cover the basics for writing collector, processor, and publisher plugins in the examples folder.
 
 
-
+## Snap Diagnostics
+Snap plugins using plugin-lib-go can be run independent of Snap to show their current running diagnostics. This diagnostic information includes:
+* Warning if dependencies not met
+* Config policy
+    * Warning if config items required and not provided
+* Collectable metrics
```

Comment:
That refers to what metrics can be collected right now. So `collectable metrics` is correct. I'll remove `metric catalog`